packages/common/components/content/blocks/query-related.marko

```diff
@@ -17,14 +21,29 @@ $ const header = input.header || 'Related Content';
             ${header}
           </@header>
         </if>
-        <@item|{ item }|>
-          <endeavor-content-block-item
-            content=item
-            image-position="right"
-            image-options={ w: 120, h: 120, fit: 'crop', crop: 'focalpoint', fpX: 0.5, fpY: 0.5 }
-            image-width=75
-            image-height=75
-          />
+        <@item|{ item, index }|>
+          <if(useNativeX(nativeX.placement, nativeX.index, site) && index === nativeX.index)>
+            $ const section = getAsObject(item, 'primarySection');
+            $ const image = getAsObject(item, 'primaryImage');
+            <endeavor-nativex-placement
+              name=nativeX.placement
+              fallback-vars={
+                url: item.canonicalPath,
+                content: { title: item.shortName, type: item.typeTitled, published: item.publishedDate },
+                image: { src: image.src, alt: image.alt },
+                section: { name: section.name },
```

Comment:
Why not just build this out inside the component and just pass content and section in?

Reply:
Better yet, why not just run everything through the nX component and have the component do the enabled check internally?
```js
$ const mods = {
  imagePosition: 'right',
  imageOptions: { w: 120, h: 120, fit: 'crop', crop: 'focalpoint', fpX: 0.5, fpY: 0.5 },
  imageWidth: 75,
  imageHeight: 75,
};
<endeavor-nativex-placement config=input.nativeX content=item section=section mods=mods>
  <endeavor-content-block-item content=item ...contentMods />
</endeavor-nativex-placement>
```

Reply:
OR, just put the nX logic inside the content-block-item component. If nX is enabled and its index matches the config, use it?